packages/components/collapse/src/Collapse.tsx

```diff
@@ -46,30 +48,26 @@ export const Collapse = ({
 
     const handleTransitionEnd = () => {
       if (current) {
-        current.style.height = 'auto';
+        if (isExpanded) {
+          current.style.removeProperty('height');
```

Comment:
Since we have `height: 0` in the default styles, when you remove the height prop, it will be collapsed (back with 0 height).

Reply:
Oh, I totally oversaw the styles file. I'm adding auto height back to the event listener.